eos/utils/options.hh

```diff
@@ -31,8 +31,7 @@
 namespace eos
 {
     /*!
-     * UnknownOptionError is thrown when no option of a given name could be
-     * found.
+     * UnknownOptionError is thrown when an Options object does contain a value for a given option key.
```

Comment:
Pretty sure this should be "does not contain a value"

Reply:
Silly me. Silly, silly me.